migrations/2_create_assets_up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE assets(
+        asset_id varchar(100) PRIMARY KEY,
```

Comment:
here we already have table name as assets so the name of primary key should be only id & if we refer this as foreign key in some other table then field name should be asset_id in foreign table. pls make these change wherever applicable.